lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -128,6 +139,28 @@ def self.download_file_from_tag(repository:, tag:, file_path:, download_folder:)
 
         download_path
       end
+
+      # Creates (or updates an existing) GitHub PR Comment
+      def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier: SecureRandom.uuid)
+        client = github_client
+        comments = client.issue_comments(project_slug, pr_number)
+
+        complete_reuse_identifier = "<!-- REUSE_ID: #{reuse_identifier} -->"
+
+        existing_comment = comments
+                           .select { |comment| comment.user.id == client.user.id } # Only match comments posted by the owner of the GitHub Token
+                           .find { |comment| comment.body.include?(complete_reuse_identifier) }
```

Comment:
I'm ok with this… but as an alternative/suggestion, it could also be a bit better to loop only once, and put both conditions in the same block:

```suggestion
        existing_comment = comments.find do |comment|
           # Only match comments posted by the owner of the GitHub Token, and with the given reuse ID
          comment.user.id == client.user.id && comment.body.include?(complete_reuse_identifier)
        end
```

Reply:
Yeah, I was back-and-forth on this – `O(n*2)` is IMHO ok for small datasets if it makes things easier to read, but since you mentioned a preference I'm fine to have my mind changed – addressed in 32fef0425dcdeb53d6b023ffbf2a4314a00bd964